packages/vega-loader/package.json

```diff
@@ -39,6 +39,6 @@
   },
   "browser": {
     "./build/vega-loader.node.js": "./build/vega-loader.min.js",
-    "./index.js": "./index.browser.js"
+    "./build/vega-loader.module.js": "./index.browser.js"
```

Comment:
Hmm, I don't think this is the full fix yet since `index.browser.js` might use features that need to be transpiled. It's a good hotfix, though.